examples/gateway/openai/README.md

```diff
@@ -0,0 +1,25 @@
+## Example route configuration for OpenAI
+
+Create or modify your MLflow configuration file. It should include a single route, for instance, to the `completions` endpoint. The structure of the configuration file should look like this:
+
+```yaml
+routes:
+  - name: completions
+    route_type: llm/v1/completions
+    model:
+      provider: openai
+      name: gpt-4
```

Comment:
```suggestion
      name: gpt-3.5-turbo
```

`gpt-3.5-turbo` is available for everyone, but gpt-4 is not.

Reply:
removed this